indexclient/client.py

```diff
@@ -554,7 +648,7 @@ def _load(self, json=None):
         json = json or self.client._get("index", self.did).json()
         # set attributes to current Document
         for k, v in json.items():
-            self.__dict__[k] = v
+            self._doc[k] = v
```

Comment:
Would it make sense to set these as class attributes instead of using getters/setters? We could define the variables in the constructor with a "zero value" then use hasattr/setattr to assign them here.

Something like...
```python
for k, v in json.items():
    if hasattr(self, k):
        setattr(self, k, v)
    else:
        logger.error("Extra data '%s' from indexd not handled by indexclient", k)
```